sklearn/semi_supervised/label_propagation.py

```diff
@@ -376,7 +383,7 @@ class LabelPropagation(BaseLabelPropagation):
     _variant = 'propagation'
 
     def __init__(self, kernel='rbf', gamma=20, n_neighbors=7,
-                 alpha=None, max_iter=30, tol=1e-3, n_jobs=1):
+                 alpha=None, max_iter=1000, tol=1e-3, n_jobs=1):
```

Comment:
I am not sure I follow so I'll rephrase what I understood: 

> If we create a separate PR just with this change, it can get included in the 0.19 final release.

Did I understand the note correctly? If so, I'll create one presently.